src/Form/matching-config/selectors-css.js

```diff
@@ -249,20 +253,23 @@ const addressCountryCode = [
 ]
 
 const birthdayDay = `
+[autocomplete=bday-day i],
 [name=bday-day i],
 [name*=birthday_day i], [name*=birthday-day i],
 [name=date_of_birth_day i], [name=date-of-birth-day i],
 [name^=birthdate_d i], [name^=birthdate-d i],
 [aria-label="birthday" i][placeholder="day" i]`
 
 const birthdayMonth = `
+[autocomplete=bday-month i],
 [name=bday-month i],
 [name*=birthday_month i], [name*=birthday-month i],
 [name=date_of_birth_month i], [name=date-of-birth-month i],
 [name^=birthdate_m i], [name^=birthdate-m i],
 select[name="mm" i]`
 
 const birthdayYear = `
+[autocomplete=bday-year i],
```

Comment:
Just realised that I didn't include these. Doesn't really matter at this stage, but was a 0 cost thing to add 🙂.